build.gradle

```diff
@@ -35,7 +35,7 @@ allprojects {
         nvaDatamodelVersion = '0.13.1'
         nvaTestUtilsVersion = "0.1.22"
         testContainersVersion = '1.15.2'
-        nvaPublicationApiVersion = '0.3.2'
+        nvaPublicationApiVersion = '0.3.3'
```

Comment:
Using the newest IonReader from nvaPublication service. This library will eventually be moved to commons if we continue to use the IonFormat. However, in the future the re-indexing process will happen from an export that is produced automatically using DynamoDbStreams and contains all resources in JsonFormat. In such case, the IonReader will not be necessary.